integration/src/test/scala/ammonite/integration/ErrorTruncationTests.scala

```diff
@@ -18,7 +18,7 @@ object ErrorTruncationTests extends TestSuite{
     val e = fansi.Str(
       intercept[ShelloutException]{ exec(file) }.result.err.string
     ).plainText
-    assert(e == expected)
```

Comment:
Also, when you do make this change, leave a comment explaining why it is necessary. This sort of special-casing is entirely non-obvious and when you come back a year later it's easy to not know why it was done